src/pages/Home/Home.test.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+
+import Home from './index';
+
+describe('Home', () => {
+  test('renders the loading correctly', () => {
+    render(<Home />);
+    expect(screen.getByText('Loading...')).toBeInTheDocument();
+  });
+  test('renders at least 1 VideoPreviewCard component', () => {
+    // render(<Home />);
```

Comment:
Question; Why is this commented?

Reply:
It's because i didn't make it works, im trying to implement in the next deliverable.